Looms.PoS/Looms.PoS.Application/Features/Order/Queries/GetOrders/GetOrdersQueryHandler.cs

```diff
@@ -0,0 +1,40 @@
+using Looms.PoS.Application.Features.Order.Queries.GetOrders;
+using Looms.PoS.Application.Interfaces.ModelsResolvers;
+using Looms.PoS.Domain.Enums;
+using Looms.PoS.Domain.Interfaces;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Looms.PoS.Application.Features.Order.Queries;
+
+public class GetOrdersQueryHandler : IRequestHandler<GetOrdersQuery, IActionResult>
+{
+    private readonly IOrdersRepository _ordersRepository;
+    private readonly IOrderModelsResolver _modelsResolver;
+
+    public GetOrdersQueryHandler(IOrdersRepository ordersRepository, IOrderModelsResolver modelsResolver)
+    {
+        _ordersRepository = ordersRepository;
+        _modelsResolver = modelsResolver;
+    }
+
+    public async Task<IActionResult> Handle(GetOrdersQuery request, CancellationToken cancellationToken)
+    {
+        var orderDaos = await _ordersRepository.GetAllAsync();
+
+        if(request.Status is not null)
+        {
+            var status = Enum.Parse<OrderStatus>(request.Status);
+            orderDaos = orderDaos.Where(x => x.Status == status).ToList();
+        }
+        if(request.UserId is not null)
+        {
+            var userId = Guid.Parse(request.UserId);
+            orderDaos = orderDaos.Where(x => x.UserId == userId).ToList();
+        }
```

Comment:
Create `GetAllOrdersFilter` record and pass these as filters to `GetAllAsync` method

Reply:
added GetAllOrdersFilter